commands/show-beacons.go

```diff
@@ -33,28 +32,31 @@ func showBeacons(c *cli.Context) error {
 	if c.String("database") == "" {
 		return cli.NewExitError("Specify a database with -d", -1)
 	}
-	res := database.InitResources("")
+	res := database.InitResources(c.String("config"))
 	res.DB.SelectDB(c.String("database"))
 
 	var data []beaconData.BeaconAnalysisView
-	cutoffScore := .7
-	if c.Bool("all") {
-		cutoffScore = 0
-	}
 
 	ssn := res.DB.Session.Copy()
-	resultsView := beacon.GetBeaconResultsView(res, ssn, cutoffScore)
+	resultsView := beacon.GetBeaconResultsView(res, ssn, 0)
```

Comment:
The latter suggestion seems to be more efficient in my opinion. The parameter exists so that cross reference analysis can pull only the "true" beacons out of the dataset. I will get  the alteration up shortly.